src/main/java/net/mcreator/workspace/misc/WorkspaceInfo.java

```diff
@@ -159,19 +159,20 @@ public Map<String, List<MItemBlock>> getCreativeTabMap() {
 		}
 
 		// Last, we add items to tabs with custom order
-		for (String tab : tabMap.keySet()) {
-			ArrayList<String> tabOrder = workspace.getCreativeTabsOrder().get(tab);
-			if (tabOrder != null) {
-				if (!tabMap.containsKey(tab))
-					tabMap.put(tab, new ArrayList<>());
+		for (Map.Entry<String, ArrayList<String>> tab : workspace.getCreativeTabsOrder().entrySet()) {
+			ModElement tabME = workspace.getModElementByName(tab.getKey());
+			if (tabME != null && tabME.getType() == ModElementType.TAB) {
+				if (!tabMap.containsKey(tab.getKey()))

```

Comment:
This doesn't render as if I reverted them :thinking:

Reply:
Should I rename `tab` to `entry` and make a `String tab` variable to further reduce amount of lines changed?

Reply:
Still not sure I fully understand, but I would say if you think the code will be clearer that way, sure, go ahead